frontend/src/components/nonprofitApply/ApplicationProcess.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { Typography, Box } from '@mui/material'
+
+
+
+
+
+function ApplicationProcess() {
+    return <div>
+
+    
+        <br></br>
+        
+        <Typography variant='h4' align="center" style={{ fontWeight: 'bold' }}>
+            Nonprofits Application Process
+        </Typography>
+
+       <br></br>
+
+       <Box sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            ml: 60,
+            mr: 60
```

Comment:
This is an unusually high margin value. if you're trying to center the content inside, I think there's better options to do it (check components/about/Value.js in develop branch for example). I think the component should be wider in general, matching the figma mockups